### ISSUE TITLE
Issue with ormsgpack.OPT_SERIALIZE_PYDANTIC in pydantic pydantic >= 2.10.0

### ISSUE THREAD

**Issue by leon11s** [aviramha/ormsgpack#311]
Hey!

After an issue in our app, I started some debugging. After some time, I found an issue with running ormsgpack==1.6.0 with pydantic >= 2.10.0.

Example:
```python
from pydantic import BaseModel
import ormsgpack

class UserUsageOverview(BaseModel):
	total_successful_referrals: int
	current_plan_name: str

usage = UserUsageOverview(total_successful_referrals=5, current_plan_name="test")
value_serialized = ormsgpack.packb(usage, option=ormsgpack.OPT_SERIALIZE_PYDANTIC)
```
The code give the following error:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Type is not msgpack serializable: UserUsageOverview

With previous versions of Pydantic, things work just fine. I tried in Python versions 3.11.9 and 3.12.7

**Comment by exg** [comment]
Hi, thanks for the report. I fixed the issue in https://github.com/aviramha/ormsgpack/pull/308 and plan to release a new version with the fix soon.

**Comment by awakim** [comment]
Hi, 

Is there an ETA for the release of this fix?

Cheers!

**Comment by talongao** [comment]
Hi,
Could you please inform me when the new version will be released?

**Comment by exg** [comment]
Sorry for the delay. I released ormsgpack 1.7.0 last week but the workflow failed to upload the wheels to PyPI because of https://github.com/PyO3/maturin/issues/2335. I have been waiting for a new release of https://github.com/pypa/gh-action-pypi-publish with a fix but, as this is taking longer than expected, I have now uploaded the wheels to PyPI manually.
